src/commonTest/kotlin/serialize/TestKeys.kt

```diff
@@ -353,5 +353,7 @@ internal class TestKeys {
         KeyVillage shouldEqual "village"
         KeySimilarQuery shouldEqual "similarQuery"
         KeyEnableABTest shouldEqual "enableABTest"
+        KeyAlternatives shouldEqual "alternatives"
+        KeyIndexLanguages shouldEqual "indexLanguages"
```

Comment:
no check on `KeyUserData`?
```suggestion
        KeyIndexLanguages shouldEqual "indexLanguages"
        KeyUserData shouldEqual "userData"
```

Reply:
It's way above, ~line 121